robottelo/api/utils.py

```diff
@@ -474,7 +476,7 @@ def wait_for_tasks(search_query, search_rate=1, max_tries=10, poll_rate=None, po
         tasks = entities.ForemanTask().search(query={'search': search_query})
         if len(tasks) > 0:
             for task in tasks:
-                task.poll(poll_rate=poll_rate, timeout=poll_timeout)
+                task.poll(poll_rate=poll_rate, timeout=poll_timeout, must_succeed=must_succeed)
```

Comment:
@pondrejk In his recent cherrypick PR https://github.com/SatelliteQE/nailgun/pull/926 in nailgun related to this in 6.12.z, and mentioned 6.12.z cpick wasn't required  as defaults are same here in robottelo, so maybe you could try rerunning PRT after removing must_succeed from wait_for_tasks. I'd wait for @pondrejk comment to confirm if its required or not?

Reply:
I need the flag to be present to make sure the task failed @Gauravtalreja1